src/expr/src/relation/func.rs

```diff
@@ -3696,9 +3718,11 @@ impl fmt::Display for TableFunc {
         match self {
             TableFunc::AclExplode => f.write_str("aclexplode"),
             TableFunc::MzAclExplode => f.write_str("mz_aclexplode"),
-            TableFunc::JsonbEach { .. } => f.write_str("jsonb_each"),
+            TableFunc::JsonbEach => f.write_str("jsonb_each"),
+            TableFunc::JsonbEachStringify => f.write_str("jsonb_each"),
             TableFunc::JsonbObjectKeys => f.write_str("jsonb_object_keys"),
-            TableFunc::JsonbArrayElements { .. } => f.write_str("jsonb_array_elements"),
+            TableFunc::JsonbArrayElements => f.write_str("jsonb_array_elements"),
+            TableFunc::JsonbArrayElementsStringify => f.write_str("jsonb_array_elements"),
```

Comment:
Could you please somehow mark it whether it's the stringify variant? I keep running into situations where I'm reading an EXPLAIN and a function call leaves out important information about the function.

E.g., `jsonb_each[stringify]`. (We put various other extra function info into brackets like this.)